assets/tests/testHelpers/selectors/components/detours/diversionPage.tsx

```diff
@@ -1,3 +1,14 @@
+import { expect } from "@jest/globals"
 import { byRole } from "testing-library-selector"
 
 export const finishDetourButton = byRole("button", { name: "Finish Detour" })
+
+export const originalRouteShape = {
+  get(container: HTMLElement): Element {
+    const maybeShape = container.querySelector(
+      ".c-detour_map--original-route-shape"
+    )
+    expect(maybeShape).not.toBeNull()
+    return maybeShape as Element
```

Comment:
Ugh, I really wish that the `expect(...).no.toBeNull()` somehow worked with TypeScript's type narrowing such that you didn't need the `as Element` on the following line.

Reply:
> Ugh, I really wish that the `expect(...).no.toBeNull()` somehow worked with TypeScript's type narrowing

:same: :1000: 💯